src/exchange/wizard-hosted-creation/add-domain-name/add-domain-name.html

```diff
@@ -0,0 +1,231 @@
+                    </div>
+                    <div class="radio">
+                        <label>
+                            <input type="radio"
+                                   data-ng-model="$ctrl.homepage.isAutoConfigurationMode"
+                                   data-ng-value="false"
+                                   data-ng-disabled="$ctrl.shouldDisabledDomainSelection">
+                            <strong data-ng-bind="$ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_domainNameSelection_ovhDomain_dnsZone_manual')"></strong>
+                        </label>
+                        <p data-ng-bind="$ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_domainNameSelection_ovhDomain_dnsZone_manual_explanation')"></p>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <!-- Non OVH domain or OVH domain managed by another NIC-->
+        <div data-ng-show="$ctrl.homepage.domainIsNotManagedByCurrentNIC">
+            <h1 class="mt-3" data-ng-bind="$ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_domainNameSelection_nonOvhDomain_header')"></h1>
+            <hr>
+
+            <p data-ng-bind-html="$ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_domainNameSelection_nonOvhDomain_explanation')"></p>
+
+            <p role="alert"
+               class="alert alert-info"
+               data-ng-bind-html="$ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_domainNameSelection_nonOvhDomain_dnsZone_explanation')"></p>
+
+            <form name="$ctrl.domainNameForm" data-ng-class="{'has-error': $ctrl.domainNameForm.$dirty && $ctrl.domainNameForm.$invalid}">
+                <label class="mt-3" for="foreignDomainName" data-ng-bind="$ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_domainNameSelection_nonOvhDomain_inputHeader')"></label>
+                <div class="row">
+                    <div class="col-md-4">
+                        <input type="text"
+                               class="form-control"
+                               id="foreignDomainName"
+                               name="foreignDomainName"
+                               placeholder="{{:: $ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_domainNameSelection_nonOvhDomain_placeholder') }}"
+                               data-ng-model="$ctrl.homepage.domainName"
+                               data-ng-change="$ctrl.reinitializeCNAMEToCheck(); $ctrl.checkIfDomainIsManagedByThisNIC();"
+                               data-ng-pattern="/^(?:[a-zA-Z0-9]+(?:[-_][a-zA-Z0-9]+)*)(?:(?:\.|\+)(?:[a-zA-Z0-9]+(?:[-_][a-zA-Z0-9]+)*))*(?:\.[a-zA-Z]{2,})$/"
+                               data-ng-disabled="$ctrl.loaders.isActivationRunning">
+                        <p data-ng-if="$ctrl.domainNameForm.foreignDomainName.$invalid && $ctrl.domainNameForm.foreignDomainName.$error.pattern">
```

Comment:
[ngMessages](https://docs.angularjs.org/api/ngMessages/directive/ngMessages) could make it more readable.